### DISCUSSION TITLE
The energy in the "./02.fp/data*/energy.raw" file does not match the energy in the "output" file.

### DISCUSSION THREAD

**Discussion by Qjump** [deepmodeling/dpdata#530] [Q&A]
The energy values extracted to the 'erergy.raw' file within the ’02.fp/data.00*' directory do not correspond with the original energy values in the 'output' files generated from CP2K.
on the left, we have the energy.raw data of a specific system from 02.fp/data*. On the right, we present the energy of the same system extracted from the "output" folder in the task* directory, multiplied by 27.211386245988031 （this is the unit conversion factor for dpdata). It's evident that energy.raw has truncated the energies of different structures to the same values（The energy difference between the first and third structures in the output is 0.01 eV, but the energy is the same in the energy.raw.）. What is the underlying cause of this phenomenon and how can we address it for improvement?
<img width="594" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/113186280/c24843ef-1498-4664-8cc9-2f786aad56d9">

**Comment by Qjump** [comment]
Is it possible that the decrease in data accuracy after fetching could be due to unit conversion? This information is from the "dpdata/cp2k/output.py" file, but I haven't been able to locate the interface for precision settings in this code.
![64941692711745_ pic](https://github.com/deepmodeling/deepmd-kit/assets/113186280/9a3021d7-cc0f-44b4-a06c-797e8bfc1103)

**Comment by wanghan-iapcm** [comment]
Please check if this PR is related https://github.com/deepmodeling/dpdata/pull/446
